PYME/recipes/processing.py

```diff
@@ -1959,3 +1959,103 @@ def execute(self, namespace):
             pass
             
         namespace[self.output_name] = res
+
+@register_module('DarkAndVarianceMap')
+class DarkAndVarianceMap(ModuleBase):
```

Comment:
Recipe module should call the refactored code in gen_sCMOS_maps (otherwise we have duplication). 

Do we need a recipe module? To what extent is this something we need to automate? For the standard full frame maps you pretty much need to take them once during the lifetime of a camera. This might change with maps for very fast frame rates on very small ROIs so I'm open to the idea of a recipe module.